dali/test/python/test_dali_tf_dataset_eager.py

```diff
@@ -224,6 +229,13 @@ def check_tf_dataset_wrong_input_type(wrong_input_datasets):
                 device_id=pipe.device_id)
         return dali_dataset
 
+
+@raises(TypeError, glob='*`input_datasets` must be a dictionary that maps input names (the `name` specified for External '
+                        'Source node in DALI pipeline) to input datasets objects (`tf.data.Dataset`) or '
+                        '`nvidia.dali.plugin.tf.experimental.Input` wrapper objects*')
```

Comment:
Likewise, you don't have to put the entire message here - and you can skip leading and trailing *.

Reply:
I shortened the patterns to the essential part